common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -121,12 +122,19 @@ async function prepareFiles(projectFolder: string, options: { browser: boolean }
 async function applyCodemods(projectFolder: string): Promise<void> {
   const project = new Project({ tsConfigFilePath: resolve(projectFolder, "tsconfig.json") });
 
+  const whiteListedFiles = [/^vitest.*\.config\.ts$/];
```

Comment:
to avoid potential policheck warnings.
```suggestion
  const skipPatterns= [/^vitest.*\.config\.ts$/];
```